ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -332,6 +332,11 @@
                    (when on-blur (on-blur event)))
         :on-request-add handle-add!
         :on-request-delete handle-del!}
+       ;; Define suggestions data element format.
```

Comment:
I don't understand these comment lines. Are they meant for users of the `:chip-input` form field type or just a reminder about the internals?

Reply:
Reminder about the internal Autocomplete component behaviour.